src/features/Content/MoviePeople/Tile/Actress/index.js

```diff
@@ -33,12 +33,29 @@ export function Actress() {
           ? (<Error />)
           : (
             <>
-              <ImagePeople src={img_base_url + api.data.profile_path} alt='' />
-              {api.data.name}
-
-              <MovieCastCrew id={id} header="Movies - cast" credits="cast"/>
-              <MovieCastCrew id={id} header="Movies - crew" credits="crew"/>
-              
+              <StyledTiles>
+                <Container>
+                  <ImagePeople src={img_base_url + api.data.profile_path} alt='' />
+                  <Information>
+                    <Title> {api.data.name} </Title>
+                    <StyledInfo>
+                      <BasicInfo>
+                        <SgColor>
+                          <DesktopInscription>Date of birth: </DesktopInscription>
+                          <MobileInscription>Birth: </MobileInscription>
+                        </SgColor>
+                        {api.data.birthday.slice(8, 10)}.
+                        {api.data.birthday.slice(5, 7)}.
+                        {api.data.birthday.slice(0, 4)}
+                      </BasicInfo>
+                      <BasicInfo><SgColor>Place of birth: </SgColor>{api.data.place_of_birth}</BasicInfo>
+                    </StyledInfo>
+                    <Biography>{api.data.biography}</Biography>
+                  </Information>
+                </Container>
+              </StyledTiles>
```

Comment:
nie myślałeś może nad wstawieniem tego do podfolderu?

Reply:
Ok poprawię to trochę później, dzięki za info